test/scala/matchers/RegexMatcherTest.scala

```diff
@@ -1,29 +1,38 @@
 package matchers
 
 import model._
-import org.scalatest._
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+import com.softwaremill.diffx.scalatest.DiffMatcher._
```

Comment:
I've not used this library before. Nice!

Reply:
The diffs without it were ... the worst 😭